src/einsteinpy/radiative_transfer.py

```diff
@@ -0,0 +1,56 @@
+import astropy.units as u
+import numpy as np
+
+from einsteinpy import constant
+from einsteinpy.coordinates import SphericalConversion
+from einsteinpy.integrators import RK45
+
+'''create a class photon which would include a position vector and a velocity vector'''
```

Comment:
This is not needed

Reply:
I am unsure of the coordinate system that might be needed , i thought spherical would be apt acc to the paper if you have any other suggestions ?